_data/changelogs/component-text-input.yml

```diff
@@ -5,6 +5,11 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
```

Comment:
@amyleadem can you update this field and merge when checks pass?

Reply:
Updated in [3ecbf7c](https://github.com/uswds/uswds-site/pull/2291/commits/3ecbf7cac1e1bb02dcd5b0399d26af246ce9cd88). Will merge when checks pass.